c/include/libsbp/ndb.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2015 Swift Navigation Inc.
```

Comment:
This is a generated file, so might need to be updated in the template. I would create a separate issue/PR to fix tho.